Content.Server/Chemistry/ReactionEffects/CreateEntityReactionEffect.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Shared.Chemistry.Reagent;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Chemistry.ReactionEffects;
+
+[DataDefinition]
+public class CreateEntityReactionEffect : ReagentEffect
+{
+    /// <summary>
+    ///     What entity to create.
+    /// </summary>
+    [DataField("entity", required: true, customTypeSerializer:typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string Entity = default!;
+
+    /// <summary>
+    ///     How many entities to create per unit reaction.
+    /// </summary>
+    [DataField("number")]
+    public uint Number = 1;
+
+    public override void Effect(ReagentEffectArgs args)
+    {
+        args.EntityManager.TryGetComponent(args.SolutionEntity, out TransformComponent transform);
```

Comment:
Just use `GetComponent` here honestly

Reply:
Just using `Get` now. I also completely forgot to check the return value anyways.